okhttp/src/main/java/okhttp3/HttpUrl.java

```diff
@@ -972,6 +975,44 @@ static HttpUrl getChecked(String url) throws MalformedURLException, UnknownHostE
     return PublicSuffixDatabase.get().getEffectiveTldPlusOne(host);
   }
 
+  public static String displayHost(Builder.DisplayMode displayMode, String host) {
```

Comment:
I don’t think DisplayMode is right as public API. It seems like an implementation detail to me.